packages/tabs/src/Tabs.tsx

```diff
@@ -99,7 +99,9 @@ function Tabs({
   as = 'button',
   ...rest
 }: TabsProps) {
-  const childrenArray = React.Children.toArray(children);
+  const childrenArray = React.Children.toArray(children) as Array<
+    React.ReactElement
```

Comment:
Should this not be `React.ReactNode`?

Reply:
We actually implicitly assume that the children are <Tab /> elements. We're currently attempting to access their props without doing any validation :(